File: src/main/java/com/partymakers/shareparty/data/persistence/party/PartyRoomPersistenceRepository.java
package com.partymakers.shareparty.data.persistence.party;

import com.partymakers.shareparty.data.persistence.party.entity.PartyRoomEntity;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface PartyRoomPersistenceRepository extends JpaRepository<PartyRoomEntity, Long> {
}

File: src/main/kotlin/com/partymakers/shareparty/data/persistence/party/PartyRoomPersistenceRepository.kt
package com.partymakers.shareparty.data.persistence.party

import com.partymakers.shareparty.data.persistence.party.entity.PartyRoomEntity
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface PartyRoomPersistenceRepository : JpaRepository<PartyRoomEntity, Long> 